service/src/main/resources/changelog/changesets/20230309_sarscov2_update_io_record_lookup_names.yaml

```diff
@@ -0,0 +1,1490 @@
+databaseChangeLog:
+  - changeSet:
+      id: "update sarscov2_nextstrain: update build_yaml, assmble_refbased record lookup and count_by_group output type."
+      author: kpiere
+      changes:
+        - update:
+            tableName: run_set
+            where: run_set_id='10000000-0000-0000-0000-000000000006'
+            columns:
+              - column:
+                  name: input_definition
+                  value: |
+                    [
+                      {
+                        "input_name": "sarscov2_nextstrain.build_name",
+                        "input_type": {
+                          "type": "primitive",
+                          "primitive_type": "String"
+                        },
+                        "source": {
+                          "type": "record_lookup",
+                          "record_attribute": "build_name"
+                        }
+                      },
+                      {
+                        "input_name": "sarscov2_nextstrain.builds_yaml",
```

Comment:
Should this and attribute name be `build_yaml`?

Reply:
The source of truth is the `nextstrain.tsv` data set in the [sample workspace](https://app.terra.bio/#workspaces/azure-featured-workspaces/COVID-19%20Surveillance%20/files), and the required workflow inputs in the [workflow script](https://raw.githubusercontent.com/broadinstitute/viral-pipelines/master/pipes/WDL/workflows/sarscov2_nextstrain.wdl). In this case, it's `build_name` and `builds_yaml`, so I think this is currently correct.